src/pages/learn/awesome_lucky/index_page.cr

```diff
@@ -94,6 +94,16 @@ class Learn::AwesomeLucky::IndexPage < PageLayout
         text: "Scribe",
         url: "https://scribe.rip/",
         description: "Alternative frontend to Medium.com"
+      mount AwesomeLink,
+        text: "DendroRithms",
+        url: "https://dendrorithms.com/",
+        description: "DendroRithms is a generative art installation inspired by tree growth rings",
+        tags: [AwesomeLink::Tags::Crypto]
```

Comment:
@wout is this a fair tag to use?

Reply:
Ah, you got there before me. :slightly_smiling_face: 
Yes, I think this is looking great!
Thanks for doing this.